Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml

```diff
@@ -1,7 +1,11 @@
 @page "/confirmation-message"
 
+@using CO.CDP.Localization
+@using Microsoft.AspNetCore.Mvc.Localization
+@inject IHtmlLocalizer<StaticTextResource> Localizer
+
 @{
-    ViewData["Title"] = "Thank you for contacting GOV.UK";
+    ViewData["Title"] = @StaticTextResource.Supplementary_Title;
```

Comment:
The @ sign on @StaticTextResource is redundant as you're already inside a code block

Reply:
Good spot, done.